src/states/node.rs

```diff
@@ -510,6 +510,9 @@ impl Node {
                 NetworkEvent::SectionInfo(ref sec_info) => {
                     self.handle_section_info_event(sec_info, our_pfx, outbox)?;
                 }
+                NetworkEvent::ExpectCandidate(ref vote) => {
+                    self.handle_expect_candidate_event(vote.clone())?
```

Comment:
Do you need to clone here or could you borrow in `handle_expect_candidate_event`?

Reply:
fixed, can move by removing ref.